### DISCUSSION TITLE
Equation numbers

### DISCUSSION THREAD

**Discussion by mklilley** [community/community#151728] [Repositories]
### Select Topic Area

Question

### Body

Does anyone know if it's possible and/or it's being considered to implement automatic equation numbering when rendering markdown files with math inside them?

I know that math rendering uses MathJax and [MathJax supports this](https://docs.mathjax.org/en/latest/input/tex/eqnumbers.html) with:
```
window.MathJax = {
  tex: {
    tags: 'all'
  }
};
```

I implemented this on my own website and it worked out of the box.

With this, one can label equations and refer to them with `label` and `ref`. If this was implemented then storing markdown files with math in a repo is really nice because one can both view the rendered contents on Github and use the same file stored on Github as input to Pandoc (that can turn markdown into e.g. latex or PDFs) without any extra processing.

**Comment by github-actions** [comment]
🕒 **Discussion Activity Reminder** 🕒

This Discussion has been labeled as dormant by an automated system for having no activity in the last 60 days. Please consider one the following actions:

1️⃣ Close as Out of Date: If the topic is no longer relevant, close the Discussion as `out of date` at the bottom of the page.

2️⃣ Provide More Information: Share additional details or context — or let the community know if you've found a solution on your own.

3️⃣ Mark a Reply as Answer: If your question has been answered by a reply, mark the most helpful reply as the solution.

Note: This dormant notification will only apply to Discussions with the `Question` label. To learn more, see our [recent announcement](https://github.com/orgs/community/discussions/70478).

Thank you for helping bring this Discussion to a resolution! 💬
